Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/coronatest/rat/profile/list/ProfileListEvent.kt

```diff
@@ -0,0 +1,6 @@
+package de.rki.coronawarnapp.ui.coronatest.rat.profile.list
+
+sealed class ProfileListEvent {
+    object NavigateToAddProfile : ProfileListEvent()
+    data class OpenProfile(val id: String, val position: Int) : ProfileListEvent()
+}
```

Comment:
The id is changing to be an Int

Reply:
Yes, but I'm waiting to merge you PR first because the id is still a string on this branch in the `Profile` data class. We'd have a conflict if I were to change it from the data class as well.